### PR TITLE
Keep username on chat

### PR DESCRIPTION
**Pull request by zyhou**
s

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ module.exports = {
             'error',
             {
                 ignoreRestSiblings: true,
+                varsIgnorePattern: '^_',
```
Comment (zyhou) [inline .eslintrc.js:27]: Now you can do `const [_, setUser] = React.useState();` without eslint error

Comment (Euregan) [reply .eslintrc.js:27]: Shouldn't it be `^_$` to prevent variables beginning with `_` ?

File: lib/useLocalStorage.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (Euregan) [inline lib/useLocalStorage.js:1]: Shouldn't hooks be in the `hooks` folder instead?

File: .eslintrc.js (hunk 1 above)
Comment (zyhou) [reply .eslintrc.js:27]: I saw some people wire _ignore or sutff like that. I prefer the underscore only too

File: lib/useLocalStorage.js (hunk 2 above)
Comment (zyhou) [reply lib/useLocalStorage.js:1]: I want to move the hooks folder inside the lib folder. What do you think?

**Comment by Euregan** [comment]
#### 📝Changed routes:


| [**/**](https://oeaki-git-keep-username.euregan.now.sh/) |
|:-:|
|<a href="https://oeaki-git-keep-username.euregan.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/VRyBypLQJETbGgXpZDYYUKOfUIM_.png">(view full size)</a> |







Commit 9d36079ce3cc9f35dd0a8f11acac24eabfb4642d (https://oeaki-git-keep-username.euregan.now.sh).

File: pages/index.js [hunk 3]
```diff
@@ -11,18 +13,19 @@ const Home = () => {
     });
     const [formErrors, setFormErrors] = React.useState({});
 
+    const [_, setUserName] = useLocalStorage('username', 'Anonymous');
```
Comment (HALLERPierre) [inline pages/index.js:16]: If an username is already present in localStorage we could display it by default

File: lib/useLocalStorage.js (hunk 2 above)
Comment (Euregan) [reply lib/useLocalStorage.js:1]: Either way's fine with me

**Pull request merged**